lib/Decompressor.js

```diff
@@ -219,10 +267,20 @@ export class Decompressor extends Transformer {
   }
 
   _transformTypedValue({entries, termType, value}) {
-    const DecoderClass = TYPE_DECODERS.get(termType);
-    const decoder = DecoderClass && DecoderClass.createDecoder(
-      {value, transformer: this});
-    if(decoder) {
+    let decoder;
+    if(this.reverseTypedLiteralTable.has(termType)) {
```

Comment:
There's a whitespace (empty line) between `let encoder; ... if(` in the compressor version of this -- we should be consistent and either delete it over there or add it here.